portfolio/src/main/java/com/google/sps/data/Message.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.data;
+
+public class Message {
+    public String comment;
+    private long author;
+    private long timestamp;
+    private String email;
+
+    // Email should be optional
+    public Message(String c, long a, long t, String e){
```

Comment:
What do you think about fully enumerating these variables (comment instead of c) and then initializing each variable like:

this.comment = comment;

Would provide more readable method signatures!